### DISCUSSION TITLE
Dialect 2 is finally in Fedora 🎉

### DISCUSSION THREAD

**Discussion by LyesSaadi** [dialect-app/dialect#279] [Show and tell]
After some long delay, I have finally been able to push [dialect 2.0.2 in Fedora](https://bodhi.fedoraproject.org/updates/FEDORA-2022-09a09304b3) :tada:!

I wanted to thank all contributors for this amazing update :tada:!

I also wanted to present my excuses for the time it took (more than a month) :sweat_smile:, as I had to introduce a new package, blueprint-compiler, in the repos.

But, now, with Dialect 2 dropping the mess python-googletrans was, Dialect is finally more stable than ever!

And I hope that all Fedora users will be able to enjoy this update as much as I am ^^!

Cheers,
Lyes Saadi

**Comment by mufeedali** [comment]
Niiice!

> for the time it took (more than a month)

You've probably noticed, but development has also slowed down as our lives got busier. So no worries :)
